admin/resources/js/stories/Select.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import Form from "../components/form/Form";
+import Select, { SelectProps } from "../components/form/Select";
+import Submit from "../components/form/Submit";
+
+export default {
+  component: Select,
+  title: "Form/Select",
+  argTypes: {
+    contextToggleHandler: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} as Meta;
+
+const TemplateSelect: Story<SelectProps> = (args) => {
+  const style = {
+    width: "25%",
+  };
```

Comment:
I totally see why you added this, but after talking with Matt, I think we should wrap our component stories in as little extra styling as possible. It's useful to know that this Select component will fill its container. And it makes Storybook's Viewport selector more useful.